docs/user_guide/source/components/engine.rst

```diff
@@ -361,13 +363,13 @@ The following table summarizes the memory contracts required by ADIOS2 engines b
 +----------+-------------+-----------------------------------------------+
 | Get      | Data Memory | Contract                                      |
 +----------+-------------+-----------------------------------------------+
-|          | Pointer     | do not modify until PerformPuts/EndStep/Close |
+|          | Pointer     | do not modify until PerformGets/EndStep/Close |
```

Comment:
Is this correct? Shouldnt it be Put?

Reply:
This is correct. There is another table for PerformPut